domain/passthrough/passthrough_grpc_client.go

```diff
@@ -168,6 +174,26 @@ func (p *passthroughGRPCClient) UserPositionsBalances(ctx context.Context, addre
 	return pooledCoins, rewardCoins, nil
 }
 
+func (p *passthroughGRPCClient) DelegationRewards(ctx context.Context, address string) (sdk.Coins, error) {
+	response, err := p.distributionClient.DelegationTotalRewards(
+		ctx,
+		&distribution.QueryDelegationTotalRewardsRequest{DelegatorAddress: address},
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	var rewardCoins = sdk.Coins{}
+	for _, v := range response.GetTotal() {
+		rewardCoins = rewardCoins.Add(sdk.Coin{
+			Denom:  v.Denom,
+			Amount: v.Amount.TruncateInt(),
+		})
+	}
```

Comment:
This is a performance optimization. `Add` resorts and validates coins on every call which is unnecessary because we know that has already been sorted by `DecCoins`.

As a result, we can simply append here which is less expensive 

```suggestion
	var rewardCoins = sdk.Coins{}
	for _, v := range response.GetTotal() {
		rewardCoins = append(rewardCoins, sdk.Coin{Denom: v.Denom, Amount: v.Amount.TruncateInt()})
	}
```

Reply:
I'm going to apply it to deploy to stage right now